src/aere_repl.erl

```diff
@@ -273,8 +273,16 @@ process_input(State, eval, I) ->
             throw(aere_error:unsupported_decl(contract));
         [Ns] when element(1, Ns) =:= namespace ->
             throw(aere_error:unsupported_decl(namespace));
-        [_|_] ->
-            throw(aere_error:unsupported_decl(multidecl))
+        Multi = [_|_] ->
+            case lists:all(fun({letfun, _, _, _, _, _}) -> true;
```

Comment:
there could be a global function to test that. Also you can use `element` so you don't depend on the number of args.

Reply:
>Also you can use element so you don't depend on the number of args.

Is that a good thing to do? What if someone mistakingly decided to use a tuple that has `letfun` as a first element for some other purpose?

Reply:
>What if someone mistakingly decided to use a tuple that has letfun as a first element for some other purpose?

I have seen this once in the code where there is a tuple `{contract, _}` and another tuple `{contract, _, _, _, _, _}` and they are used for different purposes. I think that matching on both the first atom and the number of the rest of the args makes it less likely to make mistakes.

Reply:
It doesn't make that much difference, as you wish